### PR TITLE
remove deprecations, second try

### PR DESCRIPTION
**Pull request by zsau**
#2540 
This time I paid closer attention to the spacing, while doing some minor refactoring. At least on my system, the song list looks identical. Tooltips work, too ;)

### CODE REVIEW

File: quodlibet/quodlibet/qltk/views.py [hunk 1]
```diff
@@ -257,18 +256,12 @@ def __motion(self, view, event):
         # the renderer xpad is not enough for the tooltip, especially with
         # rounded corners the label gets nearly clipped.
         MIN_HINT_X_PAD = 4
```
Comment (lazka) [inline quodlibet/quodlibet/qltk/views.py]: if you change MIN_HINT_X_PAD to 20 for example the tooltips are missaligned with this change

Comment (zsau) [reply quodlibet/quodlibet/qltk/views.py]: You're right— I didn't realize `extra_xpad` was already accounted for elsewhere. How does it look now?

File: quodlibet/quodlibet/qltk/views.py [hunk 2]
```diff
@@ -257,18 +256,11 @@ def __motion(self, view, event):
...
-        else:
-            extra_xpad = 0
+        extra_xpad = max(0, MIN_HINT_X_PAD - render_xpad)
 
-        label.set_padding(render_xpad + extra_xpad, 0)
+        # get the displayed width of the text, accounting for padding
         set_text(clabel)
-        clabel.set_padding(render_xpad, 0)
-        label_width = clabel.get_layout().get_pixel_size()[0]
-        label_width += clabel.get_layout_offsets()[0] or 0
```
[the remark this answers is not in the document]
Comment (zsau) [reply quodlibet/quodlibet/qltk/views.py]: Which themes? I must be misunderstanding something about how GTK layout works.

**Comment by declension** [comment]
Cleaning up old PRs - can we close this now, or can it be fixed?

**Pull request closed without merging**